fairchive-webapp/src/main/java/edu/harvard/iq/dataverse/validation/field/validators/PeriodoFieldValidator.java

```diff
@@ -12,16 +13,17 @@
 import org.omnifaces.cdi.Eager;
 
 import edu.harvard.iq.dataverse.persistence.dataset.ValidatableField;
-import edu.harvard.iq.dataverse.validation.PeriodoValidator;
-import edu.harvard.iq.dataverse.validation.ValidationResult;
+import edu.harvard.iq.dataverse.search.periodo.PeriodoDataFinder;
 import edu.harvard.iq.dataverse.validation.field.FieldValidationResult;
 
 @Eager
 @ApplicationScoped
 public class PeriodoFieldValidator extends MultiValueValidatorBase {
```

Comment:
Can we add some tests to this validator? We can copy something from deleted `PeriodoValidatorTest` and adjust it to use it as a test for this class